clean/hammond_pd_cprr.py

```diff
@@ -148,7 +190,31 @@ def clean():
     return df
 
 
+def clean_14():
+    df = pd.read_csv(data_file_path('raw/hammond_pd/hammond_pd_cprr_2009_2014.csv'))
+    df = df\
+        .pipe(clean_column_names)\
+        .rename(columns={
+            'date': 'investigation_start_date'
+        })\
+        .pipe(clean_dates, ['investigation_start_date'])\
+        .pipe(split_name_09)\
+        .pipe(clean_tracking_number)\
+        .pipe(clean_charges_09)\
+        .pipe(clean_disposition_09)\
+        .pipe(clean_action_09)\
+        .pipe(set_values, {
+            'agency': 'Hammond PD',
+            'data_production_year': '2021'
+        })\
+        .pipe(gen_uid, ['first_name', 'last_name', 'agency'])\
+        .pipe(gen_uid, ['uid', 'charges', 'tracking_number', 'disposition'], 'complaint_uid')
+    return df
+
+
 if __name__ == '__main__':
-    df = clean()
+    df20 = clean_20()
+    df14 = clean_14()
     ensure_data_dir('clean')
```

Comment:
FYI, we don't need to call `ensure_data_dir` ever again. Make took care of it.

Reply:
Got it. Thanks.